resources/lang/en.json

```
@@ -519,6 +519,7 @@
   },
   "map": {
     "map": "Map",
+    "random": "Random",
```

Biome (2.4.16)

[error] The key random was already declared.

(lint/suspicious/noDuplicateObjectKeys)